### ISSUE TITLE
Airtable: query functions with search conditions

### ISSUE THREAD

**Issue by kezb765** [thunkable/thunkable-issues#439]
Airtable is a great way to deliver data from an online database. There is no way to **dynamically** filter the data you want to see (there is a great way to use static filters with "views").

Why is this useful? A great way to improve airtable is working with linked fields (have a look at the introduction for linked fields "https://support.airtable.com/hc/en-us/articles/360021502354"). If you ask the content for a linked field, you do not get the data, but an ID. This ID has to be queried in the linked table and this calls for loops. It would be much easier to ask directly for the suitable row.

**The solution I'd like**
I would suggest two new functions:
- Airtable "GetRowWhere"
- Airtable "GetAllRowsWhere"
Here I decribe my idea:
"GetRowWhere" is a function for airtables. It has the following inputs:
-- rowName (for example "_id")
-- contains (for example "recAqWeert32xCxsq")
and it has the following outputs:
-- "error" and "value". Value is an object with a complete row. Error is not empty if the row does not contain the data.

"GetRowsWhere" could have the ability to use wildcards (regular expressions):
-- rowName (for example "Address")
-- filter (for example "*ain street*")
and it has the following outputs:
-- "error" and "value". Value is a list of objects with all rows that match the filter (or empty if applicable).

It could be even more interesting if we have as many inputs as columns in the table.

**Alternatives:**
As an alternative I have to filter the results from "GetAllRows" inside ThunkableX with a loop which takes time.

**Platform:**
ThunkableX

**Comment by JoeSepielli** [comment]
I agree. I migrated my app to ThunkableX and airtable from AppInventor and google fusion tables. The get rows where seems like an ideal solution. I have a kludge workaround that works for me now.
